packages/terra-table/src/TableSubheader.jsx

```diff
@@ -19,21 +22,14 @@ const TableSubheader = ({
   content,
   colSpan,
   ...customProps
-}) => {
-  const contentClassName = classNames([
-    'terra-Table-subheader',
-    customProps.className,
-  ]);
-
+}) => (
   // count is based on the number of columns and assigned in the table component which contains this subheader
-  return (
-    <tr className="terra-Table-subheaderRow">
-      <td {...customProps} className={contentClassName} colSpan={colSpan}>
-        {content}
-      </td>
-    </tr>
-  );
-};
+  <tr data-class="subheader-row">
+    <td {...customProps} className={cx('subheader', customProps.className)} colSpan={colSpan}>
```

Comment:
Oh. I see what you mean. I would assume it would use our locally scoped styling, but not 100% certain. @mhemesath @bjankord do you know the answer to Tao's question?